drivers/soft_spi/include/soft_spi_params.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2017 Markus Blechschmidt <Markus.Blechschmidt@haw-hamburg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_soft_spi
+ * @{
+ *
+ * @file
+ * @brief       Sotware SPI configuration
+ *
+ * @author      Markus Blechschmidt <Markus.Blechschmidt@haw-hamburg.de>
+ */
+
+#ifndef SOFT_SPI_CONF_H
+#define SOFT_SPI_CONF_H
+
+#include "soft_spi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Sotware SPI port descriptor array
+ */
+soft_spi_t soft_spi_config[] = {
+  //#ifdef SOFT_SPI_PARAMS_USER
```

Comment:
default configuration is empty. You should put defines above (see other drivers implementations)